### DISCUSSION TITLE
sort not working on mantine react table

### DISCUSSION THREAD

**Discussion by starxyn** [KevinVandy/mantine-react-table#415] [Q&A]
i applied sorting on mantine react table but it is still not sorted in ascending and descending order when the sort button is clicked. 
what is the proper code to sort at client-side? i also want to show initial state of column 'A' being sorted in ascending order but this initialState attribute is not working for me.

**Comment by KevinVandy** [comment]
How do you expect help without sharing your code?

**Reply by starxyn** [reply]
i only have a snippet, and not a full version.

basically, i didn't add anything special for sorting as it is enabled by default. 
`
{
  accessorKey: 'abc',
  header: 'Type',
  filterVariant: 'multi-select',
  mantineFilterMultiSelectProps: {
    data: ABC
  }
  Cells: ({ cell }) => cell.getValue<string>();
  
  const table = useMantineReactTable({
    enableSorting: true,
    .
    .
  })

 `
the thing is that when data gets sorted, the data that contains 'i' gets sorted last and appears last like this:

a
b
c
p
t
i

that's very weird.
